### PR TITLE
[Merge/Mux] Add test cases for conner cases @open sesame 11/14 15:58

### PR DESCRIPTION
**Pull request by jijoongmoon**
# PR Description

Added test cases for basepad mode with "option=0" and "option=0:0"

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: tests/nnstreamer_converter/generateGoldenTestResult.py [hunk 1]
```diff
@@ -92,6 +92,21 @@
             file.write(s[sink_1])
         file.close()
 
+    id=[0,4,7,9]
+    for i in range(0,4):
+        sink_0 = id[i]
+        with open("testsynch07_"+str(i)+".golden",'wb') as file:
+            file.write(s[i])
+            file.write(s[sink_0])
+        file.close()
```
Comment (jaeyun-jung) [inline tests/nnstreamer_converter/generateGoldenTestResult.py]: don't need file.close() when using with open ~ as file.

File: tests/nnstreamer_converter/generateGoldenTestResult.py [hunk 2]
```diff
@@ -92,6 +92,21 @@
...
+
+    for i in range(0,4):
+        sink_0 = i*3
+        with open("testsynch08_"+str(i)+".golden",'wb') as file:
+            file.write(s[i])
+            file.write(s[sink_0])
+        file.close()
```
Comment (jaeyun-jung) [inline tests/nnstreamer_converter/generateGoldenTestResult.py]: don't need file.close() when using with open ~ as file.

File: tests/nnstreamer_converter/generateGoldenTestResult.py (hunk 1 above)
Comment (jijoongmoon) [reply tests/nnstreamer_converter/generateGoldenTestResult.py]: right. thanks.

File: tests/nnstreamer_mux/runTest.sh [hunk 3]
```diff
@@ -117,4 +117,36 @@ callCompareTest testsynch05_7.log testsynch05_7.golden 13-8 "Compare 13-8" 1 0
...
+callCompareTest testsynch06_0.log testsynch05_0.golden 14-1 "Compare 14-1" 1 0
+callCompareTest testsynch06_1.log testsynch05_1.golden 14-2 "Compare 14-2" 1 0
+callCompareTest testsynch06_2.log testsynch05_2.golden 14-3 "Compare 14-3" 1 0
+callCompareTest testsynch06_3.log testsynch05_3.golden 14-4 "Compare 14-4" 1 0
+callCompareTest testsynch06_4.log testsynch05_4.golden 14-5 "Compare 14-5" 1 0
+callCompareTest testsynch06_5.log testsynch05_5.golden 14-6 "Compare 14-6" 1 0
+callCompareTest testsynch06_6.log testsynch05_6.golden 14-7 "Compare 14-7" 1 0
+callCompareTest testsynch06_7.log testsynch05_7.golden 14-8 "Compare 14-8" 1 0
+callCompareTest testsynch06_8.log testsynch05_8.golden 14-9 "Compare 14-9" 1 0
+callCompareTest testsynch06_9.log testsynch05_9.golden 14-10 "Compare 14-10" 1 0
```
Comment (myungjoo) [inline tests/nnstreamer_mux/runTest.sh]: Update argument order.
$1 should be the golden and $2 should be the log.
With the $5 == 1, the current code may say "PASS" when *.log is NULL.

**Comment by jijoongmoon** [comment]
Done. Thanks.

**Pull request merged**